c7n/cache.py

```diff
@@ -129,10 +129,11 @@ def init(self):
         self.conn = sqlite3.connect(self.cache_path)
         self.conn.execute(self.create_table)
         with self.conn as cursor:
-            log.debug('expiring stale cache entries')
-            cursor.execute(
+            result = cursor.execute(
                 'delete from c7n_cache where create_date < ?',
                 [datetime.utcnow() - timedelta(minutes=self.cache_period)])
+            if result.rowcount:
+                log.debug('expired %d stale cache entries', result.rowcount)
```

Comment:
its more just now logging when entries are actually deleted, instead of when they may be. the later is noisy and not indicative of change, the behavior now is to just log when rows are actually deleted.